packages/sdk/src/swap/types.ts

```diff
@@ -211,3 +211,10 @@ export type SwapStatusResponse =
   | DepositAddressStatusResponse
   | VaultSwapStatusResponse
   | FailedVaultSwapStatusResponse;
+
+export type BoostPoolDepth = {
+  tier: number;
```

Comment:
should we call this something like `feeTierBps`?

Reply:
i think atm we can still choose 🙂 once we exposed a name somewhere we have to keep it

Reply:
could also call it `feeBps` if that is more consistent with the other terms like `effectiveBoostFeeBps`

Reply:
Yes, lets call it `feeTierBps`. I also want to communicate the idea of tiers.

Reply:
Done